pkg/importer/importer_test.go

```diff
@@ -59,6 +59,15 @@ func TestLoadSwaggerFromTestFiles(t *testing.T) {
 	})
 }
 
+func TestLoadOpenApiFromTestFiles(t *testing.T) {
```

Comment:
Possibly target higher code coverage by splitting up the tests.

Reply:
93.8% coverage on just openapi.go from the added tests... whats your problem exactly?

Reply:
I thought of splitting them considering only one test entity at a time. Thus you have test cases for importing open-api Endpoints, Types so on and so forth. But the coverage is good as is, so it's fine.